### ISSUE TITLE
Virtualization

### ISSUE THREAD

**Issue by RealPeha** [samdenty/console-feed#53]
How about adding virtualization to the message list? This is a big performance issue.

**Comment by samdenty** [comment]
virtualization is something I'd considered but haven't had time to implement, open to accepting prs

**Comment by RealPeha** [comment]
Yes, I could do it. What do you think about [react-window](https://github.com/bvaughn/react-window)?

_Would you like to add the "hacktoberfest" topic to your repository? 🙃_

**Comment by samdenty** [comment]
react-window sounds good. sure

**Comment by yuqianma** [comment]
Any progress?
